### DISCUSSION TITLE
Updated version of OnlineAnnouncer-v2

### DISCUSSION THREAD

**Discussion by comdar16** [Pryaxis/TShock#2465] [Show and tell]
[redacted] free for use

**Comment by csmir** [comment]
This is not the path to take for announcing plugin updates, or new plugins for that matter. Please head over to https://github.com/Pryaxis/Plugins .

**Reply by comdar16** [reply]
> This is not the path to take for announcing plugin updates, or new plugins for that matter. Please head over to https://github.com/Pryaxis/Plugins .

i can't there is no Discussion button

**Reply by csmir** [reply]
Please read the repository readme properly, and submit a PR as requested at the bottom of the list.

**Reply by hakusaro** [reply]
I don’t know that I see a problem with using discussions for stuff like this, admittedly.

**Reply by comdar16** [reply]
> I don’t know that I see a problem with using discussions for stuff like this, admittedly.

Is it bad that i made posts about plugins in tshock discussion?

**Reply by hakusaro** [reply]
I dunno. Just because nobody has done it before doesn’t mean that it’s bad. We don’t have discussions enabled on the plugins repo because we were in the early beta for the feature. We could enable discussions in that repo, or keep them here? Idk. The biggest argument to having discussions enabled on plugins and not here is because we’re alerted to every new discussion (we = people who are watching this repo).

**Reply by comdar16** [reply]
> I dunno. Just because nobody has done it before doesn’t mean that it’s bad. We don’t have discussions enabled on the plugins repo because we were in the early beta for the feature. We could enable discussions in that repo, or keep them here? Idk. The biggest argument to having discussions enabled on plugins and not here is because we’re alerted to every new discussion (we = people who are watching this repo).

i think this place is good for plugin discussion too because mostly everyone is asking about stuff here not in plugin section

**Reply by csmir** [reply]
I think we should generalize where what takes place. 

There is a separate repository purely for submitting plugins and making people aware of the existence of said plugins. Splitting apart where this happens in the nature of 'discussion is discussion' doesn't seem like a valid reason to me. I think people should keep any plugin related discussion in the repository of said plugin, and submit new entries through a pull request as it was constructed initially. Discussion posts also are not moderated for validity/malware so posting plugins here seems against the strict security policy TShock sets for plugins.

TL;DR, keep plugins at plugins & main at main.

**Reply by Moneylover3246** [reply]
People talk about virtually anything in the discord discussions. I see no reason for it to not be the same in github

**Reply by csmir** [reply]
Doesn't contribute to this very issue. In Discord the discussion channel is for discussion that does not directly involve TShock. Yes, people use it for this purpose but plenty of channels specifically made for one of the above matters are already in place. The discussion feature on GitHub is more about actual TShock matters, questions regarding it and the service it provides. Discussions also exist to be marked as answered or provide some information about the residing repository, not to be announcement boxes for 3rd party additions.

**Reply by hakusaro** [reply]
@Rozen4334 do you mind taking your thoughts to a new discussion rather than discussing it in this unrelated topic?

**Reply by comdar16** [reply]
There is a category (Show and tell) i think if it exists i can post any thing that is related to tshock here marked with this category. If you can't stand it well then you have to live with it.

**Reply by comdar16** [reply]
hakusaro lock this our discussion otherwise it will be a drama which is right and which is not.

**Comment by csmir** [comment]
Following up in #2466 by request of @hakusaro.

**Comment by moisterrific** [comment]
Link redacted and conversation locked, please read [this](https://github.com/Pryaxis/TShock/discussions/2462#discussioncomment-1456398).
